modules/odc-installing-helm-charts-using-developer-perspective.adoc

```diff
@@ -5,7 +5,7 @@
 You can use either the *Developer* perspective in the web console or the CLI to select and install a chart from the Helm charts listed in the *Developer Catalog*. You can create Helm releases by installing Helm charts and see them in the *Developer* perspective of the web console.
 
 .Prerequisites
-* You have logged in to the web console and have switched to the xref:../../web_console/odc-about-developer-perspective.adoc#odc-about-developer-perspective[*Developer* perspective].
```

Comment:
Is there a reason this was removed?

Reply:
Hm, not sure how that happened since I didn't have to work on this module for the web console overview. I'll go ahead and insert the prereqs.

Reply:
Hm, not sure how that happened since I didn't have to work on this module for the web console overview. I'll go ahead and insert the prereqs.